components/homme/src/share/prim_driver_base.F90

```diff
@@ -1331,8 +1344,15 @@ subroutine prim_step_flexible(hybrid, elem, nets, nete, dt, tl, hvcoord, compute
           end if
        end if
 
-       call prim_advance_exp(elem, deriv1, hvcoord, hybrid, dt, tl, nets, nete, &
-            compute_diagnostics_it)
+       if (.not. single_column) then
+         call prim_advance_exp(elem, deriv1, hvcoord, hybrid, dt, tl, nets, nete, &
+              compute_diagnostics_it)
+       else
+         ! Update floating temperature based prescribed LS vertical velocity,
+         !  but don't update tracers at this point
+         scm_update_T = .true.
+         call prim_step_scm(elem, nets_in, nete_in, dt, tl, hvcoord, scm_update_T)
```

Comment:
Here you're in a `single_column` block, so you could just write 1, 1 if you wanted to instead of nets_in, nete_in.